src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -151,49 +151,64 @@
+(defn- render-main [{:keys [workspace curator? owner? writer? reader? bucket-access? editing? submissions-count
+                            library-schema request-refresh workspace-id storage-cost]}]
   (let [{:keys [owners]
-         {:keys [createdBy createdDate bucketName description workspace-attributes library-attributes]} :workspace} workspace]
+         {:keys [createdBy createdDate bucketName description workspace-attributes library-attributes]} :workspace} workspace
+        render-detail-box (fn [order title & children]
+                            [:div {:style {:flexBasis "50%" :order order}}
+                             (style/create-section-header title)
+                             children])]
     [:div {:style {:flex "1 1 auto" :overflow "hidden"}}
-     [:div {:style {:flex "1 1 auto" :display "flex"}}
-      [:div {:style {:flex "1 1 50%"}}
-       (style/create-section-header (str "Workspace Owner" (when (> (count owners) 1) "s")))
-       (style/create-paragraph
-         [:div {}
-          (interpose ", " owners)
-          (when owner?
-            [:span {}
-             " ("
-             (style/create-link {:text "Sharing..."
-                                 :onClick #(modal/push-modal
-                                            [AclEditor {:workspace-id workspace-id
-                                                        :request-refresh request-refresh}])})
-             ")"])])
-       (style/create-section-header "Created By")
-       (style/create-paragraph
-         [:div {} createdBy]
-         [:div {} (common/format-date createdDate)])]
-      [:div {:style {:flex "1 1 50%" :paddingLeft 10}}
-       (style/create-section-header "Google Bucket")
-       (style/create-paragraph
-         (case bucket-access?
-           nil [:div {:style {:position "absolute" :marginTop "-1.5em"}}
-                [comps/Spinner {:height "1.5ex"}]]
-           true (style/create-link {:text bucketName
-                                    :href (str moncommon/google-cloud-context bucketName "/")
-                                    :style {:color "-webkit-link" :textDecoration "underline"}
-                                    :title "Click to open the Google Cloud Storage browser for this bucket"
-                                    :target "_blank"})
-           false bucketName))
-       (style/create-section-header "Analysis Submissions")
-       (style/create-paragraph
-         (let [count-all (apply + (vals submissions-count))]
-           [:div {}
-            (str count-all " Submission" (when-not (= 1 count-all) "s"))
-            (when (pos? count-all)
-              [:ul {:style {:marginTop "0"}}
-               (for [[status subs] (sort submissions-count)]
-                 [:li {} (str subs " " status)])])]))]]
+     [:div {:style {:display "flex" :flexWrap "wrap"}}
+      (render-detail-box
+        1
```

Comment:
Because I want the row-wise layout (because width is fixed while height isn't) but I feel that the columns are more related to each other than the rows. I guess I was thinking that, if in a single column, the actual order in the document is correct while the order values allow the rendering to happen correctly across 2 columns. While it doesn't currently reflow for narrow screens, I was thinking that this at least describes the intent if/when we add support for it.